presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -322,10 +336,9 @@ public InputStreamSources createDictionaryStreamSources(Map<StreamId, Stream> st
             int rowOffset = rowGroupId * rowsInRowGroup;
             int rowsInGroup = Math.min(rowsInStripe - rowOffset, rowsInRowGroup);
             long minAverageRowBytes = columnIndexes
-                    .entrySet()
+                    .values()
```

Comment:
nit: This change looks unrelated now (although doesn't change the semantic). I think it's because we use `Map<StreamId, List<..>>` as the type for `columnIndexes`  (instead of the originally `Map<Int, Map<Int, List>>`) 

We can reorganize such changes in small refactor commits in necessary. Having huge commit mixed with small change might cause confusion when later read the history of code :)